Formula/f/flang.rb

```diff
@@ -1,8 +1,8 @@
 class Flang < Formula
   desc "LLVM Fortran Frontend"
   homepage "https://flang.llvm.org/"
-  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.8/llvm-project-20.1.8.src.tar.xz"
-  sha256 "6898f963c8e938981e6c4a302e83ec5beb4630147c7311183cf61069af16333d"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-21.1.0-rc3/llvm-project-21.1.0-rc3.src.tar.xz"
```

Comment:
Flang will need changes for new `flang-rt` runtime. Need to figure out how to handle install path as the resource-dir will be under `flang` formula rather than `llvm` formula.

Perhaps easiest way is to symlink/move libraries to `lib/` as this directory is already passed to linker. Or add more linker flags to flang.cfg (though this may require both `-L` and `-Wl,-rpath`).